tenseal/tensors/ckkstensor.py

```diff
@@ -92,6 +92,22 @@ def dot_(self, other) -> "CKKSTensor":
         self.data.dot_(other)
         return self
 
+    def mm(self, other) -> "CKKSTensor":
```

Comment:
One more thing: we should have tests for this too. 
if not in this PR, maybe open an issue, to remember to test it.

Reply:
done